### ISSUE TITLE
Logistic Frames Clientside multiplication -SevTech: Ages Server

### ISSUE THREAD

**Issue by SharkeyMyrl** [TeamPneumatic/pnc-repressurized#180]
Whenever I open the logistic menu on a frame it adds a new ghost image which can not be removed or picked up.

As seen in picture, after I accessed the logistic menu then removed the frame.
![2018-06-03_13 48 57](https://user-images.githubusercontent.com/18505028/40891116-34ec8f6e-6735-11e8-94fd-81b05b37d6c6.png)

**Comment by desht** [comment]
Yeah, I've seen this before but so far haven't been able to track it down.  There's a sync issue somewhere since the frame is gone server-side, but the client continues to render it until you log out & in again.  But it also doesn't always happen, which makes it harder to track down...

**Comment by desht** [comment]
OK, it seems opening the configuration screen is the trigger for the desync, alright:
* If I add a frame to a chest, then sneak-right-click with the configurator, its pops off and no longer displays on client
* If I add a frame, then open the config window (right-click with the configurator), close the window, *then* sneak-right-click with the configurator, the item pops off but the client continues to display the frame until relogging.

**Comment by desht** [comment]
This should be working now, and can be tested with build 200 or later from http://jenkins.k-4u.nl/job/PneumaticCraft-Repressurized/

**Comment by wrincewind** [comment]
Well, i was going to report this but it sounds like you might have solved the issue.

What if I added that it seems to sometmes ignore the 'invisible' tickbox, with some frames fading out but others staying visible, even through repeat logging in and out?

**Comment by desht** [comment]
Yeah, that's actually the same issue, also fixed.

**Comment by wrincewind** [comment]
Fantastic, thanks again!
